docs/kura-wires/multiport-wire-components/multiport-wire-components.md

```diff
@@ -8,7 +8,7 @@ With the addition of this new functionality, a compatible component instance can
 
 In the example provided, we have two components in the Wire Composer:
 
-- the **[Conditional](./conditional-component.md)** component that implements the if-then-else logic
+- the **[Conditional](../script-components/graalvm-conditional-component.md)** component that implements the if-then-else logic
```

Comment:
The GraalVM component is called "Conditional Component".

Reply:
Fixed in f32241a5b709685e44e957a4a5e56f3b52c6cd80